### ISSUE TITLE
Use latest flatted and log4js versions

### ISSUE THREAD

**Issue by hpigula** [angular/angular-cli#13724]
# 🐞 Bug report

### Command (mark with an `x`)

```
- [X ] new
- [X ] build
- [ X] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?

 NO!
 Yes, the previous version in which this bug was not present was: ....

### Description
Change to latest log4js and karma please cause there are plenty of warnings on old one

## 🔬 Minimal Reproduction
run ng new app(where app is any name)

## 🔥 Exception or Error
![screen](https://user-images.githubusercontent.com/41572588/53106837-bfd84b80-352b-11e9-855e-d5726a27edea.png)

## 🌍 Your Environment
<pre><code>
Angular CLI: 7.3.2
Node: 11.10.0
OS: win32 x64
Angular: 7.2.5
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.13.2
@angular-devkit/build-angular     0.13.2
@angular-devkit/build-optimizer   0.13.2
@angular-devkit/build-webpack     0.13.2
@angular-devkit/core              7.3.2
@angular-devkit/schematics        7.3.2
@angular/cli                      7.3.2
@ngtools/webpack                  7.3.2
@schematics/angular               7.3.2
@schematics/update                0.13.2
rxjs                              6.3.3
typescript                        3.2.4
webpack                           4.29.0
</code></pre>

**Anything else relevant?**
After installing flatted and log4js latest versions by hand no warnings on yarn build and  ng serve but when creating new app with CLI above Warning/Error shows :(

**Comment by alan-agius4** [comment]
@hpigula, if you change `karma`  to version 4 in your `package.json` it should solve the problem

I'll land a PR to fix this for newly generated workspaces.

**Comment by hpigula** [comment]
still talking about changing CircularJOSN to flatted as it is seccessor.

**Comment by alan-agius4** [comment]
Closed via b7646295fa74ea96fa370ee65d39ae6fd9816650
